### PR TITLE
Driver-history-summary-api

### PR DESCRIPTION
**Pull request by ChloeW125**
## JIRA ticket link
https://f4kblueprint.atlassian.net/jira/software/projects/F4KRP/boards/1?jql=assignee%20%3D%20712020%3Ad9f87391-0501-4d3f-bfa6-8ea683925cb5&selectedIssue=F4KRP-103

#API route to get driver history summary

## Implementation description
* API route that returns a driver's history summary, given a driver's UUID. 
* Summary includes 2 pieces of info: Total KMs driven by the driver (lifetime_kms) and the kms driven by the driver in the current calendar year (year determined by user's local comp timezone) (current_year_km)

## Steps to test
1. Works properly
2. No edge cases missed

## What should reviewers focus on?
* Handling of types and any potential edge cases

## Checklist
- [x] My PR name is descriptive and in imperative tense
- [x] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [x] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

**Review by ludavidca — APPROVED** [review]

File: backend/python/app/services/implementations/driver_history_service.py [hunk 1]
```diff
@@ -150,3 +153,40 @@ async def get_driver_history_by_year(
...
+
+            # Calculate the current year in the local timezone to determine which year to get data for
+            current_year = datetime.now(self.timezone).year
+
+            # Go through list and calculate total kms driven + record current kms driven this year
+            for entry in driver_history:
+                if current_year == entry.year:
+                    current_year_km += entry.km
+
+                total_kms += entry.km
```
[the remark this answers is not in the document]
Comment (ludavidca) [reply backend/python/app/services/implementations/driver_history_service.py]: Good to merge after you address that. LGTM!
